containerization/mdns.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/plugin.cfg b/plugin.cfg
+index 4ada8f29..08e83f84 100644
+--- a/plugin.cfg
++++ b/plugin.cfg
+@@ -35,6 +35,7 @@ log:log
+ dnstap:dnstap
+ chaos:chaos
+ loadbalance:loadbalance
++mdns:github.com/metalkube/coredns-mdns/pkg/mdns
```

Comment:
Note that this is probably going to disable caching again.

Reply:
Why? I thought that the plugins that apply are those after ours. and `cache` is just after ours.

Reply:
Oh, I guess I got it wrong :stuck_out_tongue: